gtfs-realtime/spec/en/examples/code-examples/dotnet.md

```diff
@@ -0,0 +1,23 @@
+# .NET
+To use the `gtfs-realtime-bindings` .NET classes in your own project, you need to first install the module from the [NuGet repository](https://www.nuget.org/packages/GtfsRealtimeBindings/).
+
+```
+Install-Package GtfsRealtimeBindings
+```
+
+The following code snippet demonstrates downloading a GTFS realtime data feed from a particular URL, parsing it as a `FeedMessage` (the root type of the GTFS realtime schema), and iterating over the results.
+
+```vb
+using System.Net;
+using ProtoBuf;
+using transit_realtime;
+
+WebRequest req = HttpWebRequest.Create("URL OF YOUR GTFS-REALTIME SOURCE GOES HERE");
+FeedMessage feed = Serializer.Deserialize<FeedMessage>(req.GetResponse().GetResponseStream());
+
+foreach (FeedEntity entity in feed.entity) {
```

Comment:
gtfs-realtime-bindings says this should be:

```dotnet
foreach (FeedEntity entity in feed.Entities) {
```

See https://github.com/MobilityData/gtfs-realtime-bindings/tree/master/dotnet#example-code